src/con_duct/__main__.py

```diff
@@ -359,7 +365,8 @@ def collect_sample(self) -> Optional[Sample]:
                             timestamp=datetime.now().astimezone().isoformat(),
                         ),
                     )
-        except subprocess.CalledProcessError:  # when session_id has no processes
+        except subprocess.CalledProcessError as exc:  # when session_id has no processes
+            lgr.debug("Error collecting sample: %s", str(exc))
             return None
```

Comment:
```suggestion
        except subprocess.CalledProcessError as exc: 
            lgr.debug("Could not collect sample (process may have finished): %s", str(exc))
            return None
```
I know its just debug, but since this will happen almost every run we should make it sound not scary to unsuspecting users ;)

Reply:
Ok
No need for str though

Reply:
Since its set to DEBUG, i dont see a problem leaving the exc in, it could be useful if ps fails for some other reason